larray_editor/comparator.py

```diff
@@ -43,8 +44,8 @@ def set_data(self, arrays, stack_axis):
         try:
             self.array = stack(arrays, stack_axis)
             array0 = self.array[stack_axis.i[0]]
-        except:
-            self.array = LArray([np.nan])
+        except Exception as e:
+            self.array = LArray(str(e))
```

Comment:
the idea is to display the error message in the array widget?

Reply:
Yes, because I think that is the best place to display it. Having a dialog box popping up would be annoying. The error is currently unreadable because the cell is not automatically enlarged to fit the data, but since we should do that anyway, this will be solved later automatically. The message can still be copy-pasted in that case though. Maybe it would be better for now to use LArray("failed") instead, with a comment in the code saying to change it back to LArray(str(e)) when we automatically resize columns based on the data. I tested LArray("failed") earlier and it displayed nicely (but it was much less informative for **me** as to why it failed).